src/campsite-info/components/camp-rating/camp-rating.utils.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { tooltipLabels } from './camp-rating.constants';
+import { Box } from '@mui/material';
+import { tooltipStyles } from './camp-rating.styles';
+
+export const GetLabelText = (value: number) => {
```

Comment:
only components and classes should have a leading capital letter, this should be `getLabelText`

Reply:
Is it possible that GitHub won't notice if I change an upper case letter to lower case? I applied the change, but it looks the same in the files changed on GitHub